emission/core/wrapper/confirmedplace.py

```diff
@@ -15,10 +15,8 @@ class Confirmedplace(ecwp.Place):
                   "inferred_labels": ecwb.WrapperBase.Access.WORM,
                   "expectation": ecwb.WrapperBase.Access.WORM,
```

Comment:
note that we are not supporting place-level inferences right now, so no need to include these either.